### ISSUE TITLE
Freezes every few seconds indefinitely until pressing windows button or tabbing out

### ISSUE THREAD

**Issue by candroid-man** [uberhalit/EldenRingFpsUnlockAndMore#141]
Every 5-10 seconds the game will freeze forever until I hit the windows key or tab out. I have the 144hz patch and the camera lock-on feature enabled.

**Comment by LeTrash1** [comment]
I have a similar experience, ever find a fix?

**Comment by krovjaren** [comment]
same problem, but without a camera lock-on

**Comment by LeTrash1** [comment]
> same problem, but without a camera lock-on

I think I found a work around where disabling CPU thread 0 for elden ring in task manager fixed the freezing

**Comment by krovjaren** [comment]
> > same problem, but without a camera lock-on
> 
> I think I found a work around where disabling CPU thread 0 for elden ring in task manager fixed the freezing

yeah, i think it's a right decision to fix it, also i tried to change the 3d cache size in nvidia control panel settings and tried to change some other stuff, and finally freezing is gone
